### PR TITLE
Issue 189: Add FAQ on default priors

### LINKED ISSUE #189: Prior predictive check to set defaults for more families as vignette
As part of #183 we are moving to setting default priors on more family / model distributional parameters.

For transparency and teaching, I suggest we make the prior predictive checks used to determine these values to be a vignette.

Steps:

* [ ] Create vignette template
* [ ] Add prior predictive check and determined priors for the lognormal (already in code)
* [ ] Add prior predictive check and determined priors for the gamma (new)
* [ ] Add new priors for the gamma to `epidist_prior`

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #189.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

**Comment by athowes** [comment]
Tagged for review @seabbs.

This is quite a simple PR which adds some FAQ text about the default priors. It's not perfect but it moves things forward.

File: vignettes/faq.Rmd [hunk 1]
```diff
@@ -82,4 +82,31 @@ We particularly highlight two functions which might be useful:
...
 
+## How did you choose the default priors for `epidist`?
+
+`brms` provides default priors for all parameters.
+However, some of those priors do not make sense in the context of our application.
+Instead, we used [prior predictive checking](https://mc-stan.org/docs/stan-users-guide/posterior-predictive-checks.html) to set `epidist`-specific default priors which produce epidemiological delay distribution mean and standard deviation parameters in a reasonable range.
+
+For example, here are summary statistics for the distributions on the delay distribution mean and standard deviation, as implied by the following prior distributions on the `brms` lognormal `mu` and `sigma` parameters
+$$
+\mu \sim \mathcal{N}(2, 0.5^2), \\
```
Comment (seabbs) [inline vignettes/faq.Rmd:93]: I'm not convinced we want to have an additional point of truth for our priors we want to keep updated but happy to keep for now @athowes

File: vignettes/faq.Rmd [hunk 2]
```diff
@@ -82,4 +82,31 @@ We particularly highlight two functions which might be useful:
...
+```{r message=FALSE, results='hide'}
+set.seed(1)
+fit_ppc <- epidist::epidist(
+  data = data, family = brms::lognormal(), sample_prior = "only", seed = 1
+)
+```
+
+```{r}
+draws <- extract_lognormal_draws(fit_ppc)
+summary(draws$mean)
```
Comment (seabbs) [inline vignettes/faq.Rmd:106]: might be better to make a plot?

**Review by seabbs — APPROVED** [review]
This is great. A few minor comments we can address in follow up

File: vignettes/faq.Rmd (hunk 2 above)
Comment (athowes) [reply vignettes/faq.Rmd:106]: It would be better but the plot is hard to make because of the outliers

**Pull request merged**